AnaProd/HH_bbWW/baseline.py

```diff
@@ -67,17 +70,24 @@ def RecoHWWCandidateSelection(df):
                                    if (N_sel_Mu == 1 && N_sel_Ele == 1) return 0;
                                    return -1; """)
     return df
-
-
 def RecoHWWJetSelection(df):
-    df = df.Define("Jet_Incl", f"v_ops::pt(Jet_p4)>20 && abs(v_ops::eta(Jet_p4)) < 2.5 && ( Jet_jetId & 2 )")
+    df = df.Define("Jet_Incl", f"v_ops::pt(Jet_p4)>20 && abs(v_ops::eta(Jet_p4)) < 2.4 && ( Jet_jetId & 2 )")
```

Comment:
keep 2.5, it is the recommended cut for btag validity, we can apply a tighter cut at the final selection level